units/systemd-journal-upload.service.in

```diff
@@ -14,6 +14,7 @@ After=network-online.target
 [Service]
 ExecStart=@rootlibexecdir@/systemd-journal-upload --save-state
 User=systemd-journal-upload
+Group=systemd-journal-upload
```

Comment:
hmm, we should probably try to minimize our unit files generally, and drop redundant settings. Hence: we probably should remove it from the other units too, rather than add it here ;-)